sparta/sparta/memory/BlockingMemoryIF.hpp

```diff
@@ -9,13 +9,13 @@
 #include "sparta/memory/MemoryExceptions.hpp"
 #include "sparta/memory/AddressTypes.hpp"
 #include "sparta/memory/DebugMemoryIF.hpp"
-#include "sparta/memory/DMI.hpp"
 
 namespace sparta
 {
     namespace memory
     {
         class TranslationIF;
+        class DMIBlockingMemoryIF;
 
         /*!
          * \brief Pure-virtual memory interface which represents a simple,
```

Comment:
This comment needs to be updated since this class isn't pure virtual.

Reply:
Hmmm... the methods `try[Read,Write]` are pure virtual...